src/chapters/sql-part1/queries.rst

```diff
@@ -0,0 +1,256 @@
+SQL Queries
+===========
+
+.. index:: ! query, ! CRUD, ! result set
+
+When we want to perform an operation on a database, we write a SQL **query**.
+Queries can vary widely in complexity and purpose depending on the operation we need to perform.
+Each query produces a **result set** containing the requested data.
+Ultimately, we can boil down each query's purpose into one of the following four categories:
+
+#. Create
+#. Read
+#. Update
+#. Delete
+
+**CRUD** or *Create*, *Read*, *Update*, *Delete* represents the four major operations we perform when we work with data.
+Let's see how we can do this with SQL!
+
+.. admonition:: Note
+
+   Throughout the rest of the chapter, we will be using an example of an events database.
+   Our event planner, Mary, has a database storing the guest info of every event she has ever planned, as well as event info about each event.
+   She named each table of guest info after the event itself.
+
+Create
+------
+
+When we talk about creating something in SQL, we could want to perform several operations.
+We may want to create a table, add a row, or add a column.
+
+Creating a Table
+^^^^^^^^^^^^^^^^
+
+Creating a Table from Scratch
+"""""""""""""""""""""""""""""
+
+Let's say that we are working on our events database.
+Mary has been working hard planning the Li wedding.
+She wants us to create a table with the pertinent guest info of every guest invited.
+Each row will represent one guest.
+The columns include the unique number id for each guest, the guest's first name, their last name, their dietary preferences, and whether or not they will be in attendance.
+
+To add a table to our database for the Li wedding, we need to use the ``CREATE TABLE`` command.
+
+.. sourcecode:: mysql
+   :linenos:
+
+   CREATE TABLE LiWedding (
+      GuestID int,
+      LastName varchar(255),
+      FirstName varchar(255),
+      Attending BOOL,
+      Diet varchar(255)  
+   );
+
```

Comment:
FYI: I did a little research online for SQL naming conventions, and while each person/company has different preferences, they all agreed on a "lowercase_underscore" system for names.

I used this style for the exercises, and I will update the studio to match.

Reply:
Done! Thanks!